modules/audigentRtdProvider.md

```diff
@@ -1,76 +1,82 @@
 ## Audigent Real-time Data Submodule
 
-Audigent is a next-generation data management platform and a first-of-a-kind 
-"data agency" containing some of the most exclusive content-consuming audiences 
+Audigent is a next-generation data management platform and a first-of-a-kind
+"data agency" containing some of the most exclusive content-consuming audiences
 across desktop, mobile and social platforms.
 
-This real-time data module provides quality user segmentation that can be 
-attached to bid request objects destined for different SSPs in order to optimize 
-targeting. Audigent maintains a large database of first-party Tradedesk Unified 
-ID, Audigent Halo ID and other id provider mappings to various third-party 
+This real-time data module provides quality user segmentation that can be
+attached to bid request objects destined for different SSPs in order to optimize
+targeting. Audigent maintains a large database of first-party Tradedesk Unified
+ID, Audigent Halo ID and other id provider mappings to various third-party
 segment types that are utilizable across different SSPs.  With this module,
 these segments can be retrieved and supplied to the SSP in real-time during
 the bid request cycle.
 
-### Usage
+### Publisher Usage
 
 Compile the audigent RTD module into your Prebid build:
 
-`gulp build --modules=userId,unifiedIdSystem,rtdModule,audigentRtdProvider,rubiconBidAdapter`
+`gulp build --modules=userId,unifiedIdSystem,rtdModule,audigentRtdProvider,appnexusBidAdapter`
+
+Add the Audigent RTD provider to your Prebid config.  For any adapters
+that you would like to retrieve segments for, add a mapping in the 'mapSegments'
+parameter.  In this example we will configure publisher 1234 to retrieve
+appnexus segments from Audigent. See the "Parameter Descriptions" below for
+more detailed information of the configuration parameters.
 
-Configure Prebid to add the Audigent RTD Segment Handler:
 ```
 pbjs.setConfig(
-	...
+    ...
     realTimeData: {
-        auctionDelay: 1000,
+        auctionDelay: auctionDelay,
         dataProviders: [
-        	{
-        		name: "audigent",
-        		waitForIt: true
-        	}
+            {
+                name: "audigent",
+                waitForIt: true,
+                params: {
+                    mapSegments: {
+                        'appnexus': true,
+                    },
+                    segmentCache: false,
+                    requestParams: {
+                        publisherId: 1234
+                    }
+                }
+            }
         ]
     }
     ...
 }
 ```
 
-Audigent segments will then be attached to each bid request objects in
-`bid.realTimeData.audigent_segments`
+### Parameter Descriptions for the Audigent `dataProviders` Configuration Section
 
-The format of the segments is a per-SSP mapping:
+params.mapSegments | Required | Object
```

Comment:
The formatting of this table needs help. I'll fix it over on the docs PR and then you can copy here if you'd like.

Reply:
I fixed the doc table already, I'll copy it over here.